site/docs/build-ref.html

```diff
@@ -167,82 +168,78 @@ <h3 id="labels">Labels</h3>
 </pre>
 
 <p>
-  In the typical case that a label refers to the same repository it occurs
-  in, the repository name may be left out. So, inside <code>@myrepo</code>
-  this label is usually written as
+  The first part of the label is the repository name, <code>@myrepo//</code>.
+  In the typical case that a label refers to the same repository it is used
+  in, the repository identifier may be abbreviated as <code>//</code>. So, inside 
+  <code>@myrepo</code> this label is usually written as
 </p>
 
 <pre>
 //my/app/main:app_binary
 </pre>
 
 <p>
-
-  Each label has two parts, a package name (<code>my/app/main</code>)
-  and a target name (<code>app_binary</code>). Every label uniquely
-  identifies a target. Labels sometimes appear in other forms; when
-  the colon is omitted, the target name is assumed to be the same as
-  the last component of the package name, so these two labels are
-  equivalent:
+  The second part of the label is the un-qualified package name
+  <code>my/app/main</code>, the path to the package
+  relative to the repository root.  When the label refers to the same
+  package it is used in, the package name (and optionally, the colon)
+  may be omitted.  So, inside <code>@myrepo//my/app/main</code>,
+  this label may be written either of the following ways:
 </p>
 
 <pre>
-//my/app
-//my/app:app
+app_binary
+:app_binary
 </pre>
 
 <p>
-  Short-form labels such as <code>//my/app</code> are not to
-  be confused with package names. Labels start with <code>//</code>,
-  but package names never do, thus <code>my/app</code> is the
-  package containing <code>//my/app</code>.
-
-  (A common misconception is that <code>//my/app</code> refers
-  to a package, or to <em>all</em> the targets in a package; neither
-  is true.)
+  (It is a matter of convention that the colon is omitted for files,
+  but retained for rules, but it is not otherwise significant.)
```

Comment:
```suggestion
  It is a matter of convention that the colon is omitted for files,
  but retained for rules, but it is not otherwise significant.
```

Reply:
Click and drag over the line numbers, then hit the little blue "+" tooltip, and you can make a multiline suggestion.